### DISCUSSION TITLE
npmuser: no such user

### DISCUSSION THREAD

**Discussion by pinfold** [NginxProxyManager/nginx-proxy-manager#2767] [Q&A]
I am running nginx in a docker container on a Synology NAS. It has been running perfectly for a year. Yesterday about noon it stopped working. The log is as follows:
    
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service prepare: starting
❯ Configuring npmuser ...
id: 'npmuser': no such user
useradd: UID 0 is not unique
s6-rc: warning: unable to start service prepare: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.

Can anyone help me understand what changed and how to correct it? I used Portainer following stack file

version: '2.9'
services:
  app:
    image: 'jc21/nginx-proxy-manager:latest'
    ports:
      - '8080:80'
      - '8181:81'
      - '4443:443'
    volumes:
      - /volume1/docker/nginx/config.json:/app/config/production.json
      - /volume1/docker/nginx/data:/data
      - /volume1/docker/nginx/letsencrypt:/etc/letsencrypt
    environment:
      - PUID=0
      - PGID=0
      - TZ=America/Los_Angeles

**Comment by pinfold** [comment]
I was able to trace this back to a watchtower update of the latest release. I chose to update the old stack, move the permissions over to my UID and try relaunching. The system persistently asked for npmuser. The only way to fix this for me was to step back to github-pr-2453. I am now back up and running but if someone can tell me what is causing this problem so I can get back on the latest release I would be grateful.

**Comment by gummysock** [comment]
I couldn't figure out a fix for this issue either, had to roll back some versions.
